lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
Also, the other thing is you'll want at some point to list the keys of the registry to make a menu in some u.i. of possible harvester types and then you'd have to convert them to strings.